caniusepython3/dependencies.py

```diff
@@ -78,7 +78,12 @@ def blocking_dependencies(projects, py3_projects):
     evaluated = set()
     for project in projects:
         log.info('Checking top-level project: {0} ...'.format(project))
-        dist = distlib.locators.locate(project)
+        try:
+            dist = distlib.locators.locate(project)
+        except AttributeError:
+            # This is a work around //bitbucket.org/pypa/distlib/issue/59/
```

Comment:
When the sentence ends in a url do you prefer 
   # This is a work around //bitbucket.org/pypa/distlib/issue/59/. 
or
  # This is a work around //bitbucket.org/pypa/distlib/issue/59/ .
or
  # This is a work around. //bitbucket.org/pypa/distlib/issue/59/